### PR TITLE
Clean

### PR DESCRIPTION
**Pull request by samolds**
cleaned up code and examples for codebender submission

### CODE REVIEW

File: ArdusatLogging.cpp [hunk 1]
```diff
@@ -1,13 +1,31 @@
...
+ *   Organization:  Ardusat
+ *        Version:  2.0
+ *       Compiler:  Arduino
+ *
+ *        Created:  12/3/2014
+ *         Author:  Ben Peters (ben@ardusat.com)
+ *   Last Updated:  8/25/2015
+ *   Last Updater:  Sam Olds (sam@ardusat.com)
+ *
+ * =====================================================================================
```
Comment (bencpeters) [inline ArdusatLogging.cpp]: Let's leave this as doxygen-style

File: README.md [hunk 2]
```diff
@@ -1,18 +1,17 @@
...
-the [Ardusat SDK](http://github.com/ardusat/ardusatsdk) to be included as well.
+the [Ardusat SDK](http://github.com/ardusat/ardusatsdk) to be installed as well.
 
-# Installing the Logging SDK
-Installing the SDK is easy - it works like any other third party Arduino library. Just download 
-the SDK at https://s3-us-west-2.amazonaws.com/ardusatweb/ArdusatSDK-Logging.zip or clone this repository
-to your hard drive, then open the Arduino IDE, go to Sketch -> Import Library -> Add Library and
+## Installing the Logging SDK
+Installing the SDK is easy - it works like any other third party Arduino library. Just [download 
+the SDK](https://s3-us-west-2.amazonaws.com/ardusatweb/ArdusatSDK-Logging.zip) or clone this repository
```
Comment (bencpeters) [inline README.md]: this link should be https://www.ardusat.com/downloads/ArdusatSDK-Logging.zip

File: README.md [hunk 3]
```diff
@@ -168,9 +167,9 @@ each reading, which can be used to calculate the total amount of space required
 Every data log has an identical header that identifies the sensor, the type of sensor, and the
 timestamp the measurement was taken.
 ```
-uint8_t type;
-uint8_t id;
-uint32_t timestamp;
+unsigned char type;
+unsigned char id;
+unsigned int timestamp;
```
Comment (bencpeters) [inline README.md]: `unsigned long`

File: examples/sd_card/sd_card.ino [hunk 4]
```diff
@@ -23,13 +45,13 @@
...
+const short READ_INTERVAL = 10000; // interval, in ms, to wait between readings
 
-// These pins are used for measurements
-#define UVOUT A0      // Output from the UV sensor
-#define REF_3V3 A1    // 3.3V power on the Arduino board
-#define SD_CS_PIN 10  // CS pin used for SD card reader. Reader should be wired to DIO
-                      // 10, 11, 12, 13
+// Pins used for measurements
+const short UVOUT = A0;      // Output from the UV sensor
+const short REF_3V3 = A1;    // 3.3V power on the Arduino board
```
Comment (bencpeters) [inline examples/sd_card/sd_card.ino]: these values aren't relevant to logging - did they get removed from the main repo?

File: README.md (hunk 3 above)
Comment (samolds) [inline README.md]: nice catch

File: examples/sd_card/sd_card.ino (hunk 4 above)
Comment (samolds) [inline examples/sd_card/sd_card.ino]: Those must have accidentally got copied over.

**Pull request closed without merging**